persistence-worker/main.py

```
@@ -0,0 +1,134 @@
+                ]
+                await conn.copy_records_to_table(
+                    "stock_ticks",
+                    columns=("time", "symbol", "stock_code", "price", "volume"),
+                    records=records,
+                )
+            print(f"Saved {len(current_batch)} rows to database")
+        except Exception as e:
+            print(f"Error saving to database: {e}")
+
+    async def run(self):
+        print(f"[INIT] Starting PersistenceWorker...")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)